app/models/section.rb

```diff
@@ -40,9 +40,7 @@ def build_section
     empty_section!
```

Comment:
This seems really expensive - you make one change to the order of the layout file, and all of the lessons get deleted and re-built. Is there another way to do this where you only update the order of the lessons and create new lessons? Any changes to existing lessons should be updated by the webhook for those lessons.

Reply:
True. But the way stuff is built in LHTP, it would be a ton of work to modify and re-order everything to conform to the new layout file. I'll add this to the long-term to do list for this project tho.